ci/date-check/src/main.rs

```diff
@@ -124,9 +124,7 @@ fn filter_dates(
 }
 
 fn main() {
-    let root_dir = env::args()
-        .nth(1)
-        .expect("expect root Markdown directory as CLI argument");
+    let root_dir = env::args().nth(1).unwrap_or(".".into());
```

Comment:
I'd rather not make this change because then people doing `cargo run` in date-check's directory will get weird results and not realize why.